NewRepo.md

```diff
@@ -0,0 +1,28 @@
+Students are welcome to add new repositories to our Organization. Fill out this form https://goo.gl/forms/qJ2wXrD7UlKc0V8V2 to add your repository. 
+
+You should follow the guidelines stated below. Please read this carefully before you fill out the form.
+
+## Guidelines for maintainers
+1. The student should be enrolled in NIT Durgapur for the current Academic session when he adds the repository.
+2. Join our [gitter channel](https://gitter.im/NIT-dgp/General), [mailing list](https://groups.google.com/forum/?hl=en#!forum/nitdopensource) and [facebook group](https://www.facebook.com/groups/NITDgpOS).
+3. The student should have adequate involvement and contribution in our community.
+4. The student should abide by the [Code of Conduct](https://github.com/NIT-dgp/Guidelines/blob/master/CODE_OF_CONDUCT.md). Any act of misconduct shall be dealt strongly and NIT-dgp holds the right to remove that person from the Organization.
+5. The student should be ready to take enough responsibilities as the maintainer of the repository and help those who try to contribute to the repository.
+6. The student should be available in the gitter channel in case of queries regarding the repository.
+
+## Guidelines for repositories
+1. We expect repositories in our organization help students learn about open source and help them in their Open source journey. So it should have adequate scope of learning.
+2. Repositories should have adequate number of issues. At least 10 open issues and 3 newcomer issues will be good to start with.
+3. The maintainer should decide on a suitable open source License for the repository.
+
+## How to Add an existing repository
+After you fill out this form we will add you to our organization after reviewing your project and involvement. Once you have the permissions it's just few steps to add your repository.
+
+1. Go to the `Settings` tab of your repository
+![settings](https://cloud.githubusercontent.com/assets/10486343/22163632/978dd1f4-df79-11e6-8413-25124024cfbc.png)
+2. Scroll down to the `Danger Zone` section and press the `Transfer` button
+![transfer](https://cloud.githubusercontent.com/assets/10486343/22163645/a19ad584-df79-11e6-9c90-5336a8dcb183.png)
+3. Enter you repository name and the organization name as `NIT-dgp`
+![confirm](https://cloud.githubusercontent.com/assets/10486343/22163648/a83d49b2-df79-11e6-94ed-b54be5fc6f88.png)
+
+Voila! You are done. The repository will be transferred to  our organization. The url of the repository changes in this process. It is better to update the remote urls in your local repository as well. See [Transferring a repository](https://help.github.com/articles/about-repository-transfers/) in Github Help.
```

Comment:
I think it would be useful to add that their issues and PRs will be transferred as well. We were confused in the beginning :sweat_smile: